Assets/Scripts/SketchControlsScript.cs

```diff
@@ -724,15 +724,6 @@ public bool InGrabCanvasMode
         {
             get
             {
-#if (UNITY_EDITOR || EXPERIMENTAL_ENABLED)
-                if (Config.IsExperimental)
-                {
-                    if (App.Scene.ActiveCanvas != App.Scene.MainCanvas)
-                    {
-                        return true;
-                    }
-                }
-#endif
                 return false;
```

Comment:
With this change, isn't `InGrabCanvasMode` always false? If so, the added code below never runs.

Reply:
Yeah. I was thinking the same myself. At the time I was erring on the side of "least changes" but InGrabCanvasMode and the conditional that uses it do nothing now.

Reply:
if it's not used anywhere else, I'd suggest that we just get rid of it, unless you could see it being used elsewhere later? Though, if it breaks things, perhaps it's best to remove now and come back to the idea entirely when we *need* to :D